cvmfs/magic_xattr.cc

```diff
@@ -72,6 +74,13 @@ MagicXattrManager::MagicXattrManager(
 
   Register("user.authz", new AuthzMagicXattr());
   Register("user.external_url", new ExternalURLMagicXattr());
+
+  Register("user.cleanup_unused_first", new CleanupUnusedFirstMagicXattr());
+  Register("user.list_open_hashes", new ListOpenHashesMagicXattr());
+  Register("user.list_groups_open_hashes",
+           new ListGroupsOpenHashesMagicXattr());
+  Register("user.list_managed_mountpoints",
```

Comment:
Same here, having this as an xattr for a specific repository is a bit unexpected, I would also make this a cvmfs_talk command